index.js

```diff
@@ -1,118 +1,235 @@
 /************************************************************** Task 1: Warm-up! **************************************************************/
 //Task a: declare a variable called votingAge, console log true if age > 18 (no function required)
+let votingAge=20;
+if(votingAge > 18){
+    console.log(true)
+}
 
 
-
+// What I want to answer
 
 
 //Task b: declare a variable and then use a conditional to change the value of that variable based on the value assigned to a second variable (no function required)
 
-
-
+let dog = "Sofie";
+let dogFur = "red";
+if (dogFur != "Fluffy Black & white"){
+    dog = "Defintely not Sofie";
+}
+console.log (dog)
 
 
 //Task c: Convert string ("1999") to integer (1999)  (no function required) // hint look up the Number method
 
+console.log(parseInt("1999"));
```

Comment:
This is the correct way to do it. Good practice is to always include the radix (the base of the input number, in this case `10`). `console.log(parseInt("1999", 10));` (similarly, you can use this to parse hexadecimal (16) binary (8) etc).
The method they were looking for is `console.log(Number("1999"));`. Though this works, there are some reasons (constructors, which you haven't learned yet) why this is a potentially very confusing way.
There are other ways when performance is important, but no need to confuse the issue yet. =P